database/seeders/AnnouncementSeeder.php

```diff
@@ -14,13 +18,39 @@ class AnnouncementSeeder extends Seeder
      */
     public function run()
     {
+        $admin_id = null;
+        //Create an entry for Super_Admin if not present, else fetch the id for the existing one.
+        if(User::count() == 0){
+            $admin = User::create([
+                'first_name'     => "Super admin",
+                'last_name'     => "User",
+                'email'    => 'superadmin@email.com',
+                'password' => bcrypt('secret'),
+                'email_verified_at' => Carbon::now()
+            ]);
+
+            $admin->ownedTeams()->save(Team::forceCreate([
+                'user_id' => $admin->id,
+                'name' => explode(' ', $admin->first_name, 2)[0]."'s Team",
+                'personal_team' => boolval(1),
+            ]));
+    
+            $admin->assignRole('super-admin');
+            $admin_id = $admin->id;
+        } 
+        else {
+            $admin = User::where('email', 'superadmin@email.com')->first();
```

Comment:
We should get the first user with the role "super admin" and assign it instead of email matching. People tend to change seeders and often the link between the user seeder and this one might change